ocaml/idl/datamodel_vm.ml

```diff
@@ -1331,6 +1331,9 @@ let set_HVM_boot_policy = call ~flags:[`Session]
            namespace ~name:"PV" ~contents:pv ();
            namespace ~name:"HVM" ~contents:hvm ();
            field ~ty:(Map(String, String)) "platform" "platform-specific configuration";
+           field ~lifecycle:[Prototyped, rel_naples, ""] ~ty:(Map(String, String)) "NVRAM"
```

Comment:
We normally put new fields at the end to avoid potential problems with compatibility if the constructor takes positional arguments.

Also, I believe we were going to make this read-only and add an explicit setter that is blocked if the power state is not halted?

Reply:
I'll move it. The read-only+setter is going to be part of the next PR.